public/assets/locates/ar/translation.json

```diff
@@ -0,0 +1,3 @@
+{
+    "Welcome_to_React": "مرحبًا بك في React"
+}
```

Comment:
nice structure, we will do stuff like that but we dont need this translation really. So you can delete this and keep the file empty. Other thing is, you have a typo at the folder name. It should be `locales` not `locates` .